jetty-documentation/src/main/asciidoc/reference/jetty-xml/jetty-xml-syntax.adoc

```diff
@@ -53,7 +53,7 @@ The following XML configuration file creates some Java objects and sets some att
       <Set name="name">demo2</Set>
     </Get>
   </Ref>
-</Configure>      
+</Configure>
 ----
 
 The XML above is equivalent to the following Java code:
```

Comment:
We used to have the [fileserver example](https://wiki.eclipse.org/Jetty/Tutorial/Embedding_Jetty#Configuring_a_File_Server) in the embedded section, which I think was a good example as it is more on topic.   Perhaps that example could be included here or in this section?